monadic_expected/monadic_functions_for_expected.org

```diff
@@ -213,7 +215,22 @@ After ~value_or~ functions add the new functions:
   // [expected.object.swap], swap
 #+END_SRC
 
-** Add a new sub-clause Monadic operations for [expected.monadic] between [expected.observe] and [expected.equality]   
+** Add following at the end [expected.object.observe] Observers:
+
+ 1 ~template <class G = E> constexpr E error_r(G&& e) const&;~
```

Comment:
Presume the intent is error_or

Reply:
Yes

Reply:
ok, let me pull it and check the format and such -- otherwise it seems fine